dist/offline_installer/debian/build_offline_installer.sh

```diff
@@ -118,4 +115,14 @@ sudo chroot build/chroot env DEBIAN_FRONTEND=noninteractive apt-get install -d -
 mkdir -p build/offline_installer/debs
 cp dist/offline_installer/debian/header build/offline_installer
 cp build/chroot/var/cache/apt/archives/*.deb build/offline_installer/debs
-(cd build; makeself offline_installer scylla_offline_installer.sh "Scylla offline package" ./header)
+cd build
+if [ ! -x makeself-$MAKESELFVER.run ]; then
+    wget https://github.com/megastep/makeself/releases/download/release-$MAKESELFVER/makeself-$MAKESELFVER.run
+    chmod a+x makeself-$MAKESELFVER.run
+fi
```

Comment:
Why? Fedora 31 has makeself-2.4.0.

Reply:
Let's add it to install-dependencies instead. Please post it as a separate patch, since I need to regenerate the toolchain.

Reply:
@avikivity this is because build_offline_install.sh only works in target distribution for now, and CentOS7 does not have makeself-2.4.0.
For same reason, it cannot be add on install-dependencies.
We probably can make build script to use docker like dbuild, anyway.

Reply:
Oh, I mentioned "CentOS7" but it's on Debian/Ubuntu, sorry. Older Debian distribution does not have makeself-2.4.0.

Reply:
Let's run it in dbuild then. I don't want the build system to download random things.

Reply:
I see it uses debootstrap/chroot, so it should be able to run it from Fedora dbuild. We'll have to add debootstrap to install-dependencies too.

Reply:
Yeah, we need builds to remain reproducible (think enterprise release lifecycle of many years), and downloading tools breaks this.